pkg/api/latest/latest.go

```diff
@@ -15,13 +15,13 @@ var Version = unversioned.GroupVersion{Group: "", Version: "v1"}
 
 // OldestVersion is the string that represents the oldest server version supported,
 // for client code that wants to hardcode the lowest common denominator.
-var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1beta3"}
+var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
```

Comment:
`pkg/api/latest/tags_test.go#TestDescriptions` skips test based on that, I don't think we should be skipping that test for v1, though. Other than that I don't see us using that variable, so I'd propose ditch it here. We'll add it when it'll be necessary.